src/components/Blogs/index.jsx

```diff
@@ -3,24 +3,45 @@ import { Container, Row } from "react-bootstrap";
 import SearchBar from "../SearchBar";
 import BlogCard from "../BlogCard";
 import "./index.scss";
+import i18next from "i18next";
 
-const POSTS_PER_PAGE = 9;
+const POSTS_PER_PAGE = 18;
 const BLOG_API_URL = `https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts?per_page=${POSTS_PER_PAGE}`;
+const EN_REFERENCE_NUMBER = "10221";
+const AR_REFERENCE_NUMBER = "38299";
+// numbers "10221" and "38299" are the refernce numbers from word press that we get after taging each post to its language in order to make the change between the languages
```

Comment:
comments go above what they are explaining.

you dont need line 13/14, delete those

Reply:
can you please take a look now?